src/utils/index.ts

```diff
@@ -394,7 +394,7 @@ export const detectInjectedType = (): 'COIN98' | 'BRAVE' | 'METAMASK' | 'COINBAS
   const isMetamask = !!ethereum?.isMetaMask
   const isCoin98 = isMetamask && ethereum.isCoin98
   const isBraveBrowser = checkForBraveBrowser()
-  const isCoinbase = window.ethereum?.selectedProvider?.isCoinbaseWallet || window.ethereum?.providers
+  const isCoinbase = ethereum?.isCoinbaseWallet
```

Comment:
This might not corrected. Have you tested it clearly?

Reply:
sure, Trang also tested

Reply:
but this part have some issue when user try to use many wallets, it will alway priority coinbase wallet, need to refactor it